src/renderers.js

```diff
@@ -0,0 +1,15 @@
+const glVectorRenderers = [
+  { name: 'Mapbox GL JS', value: 'mapbox-gl' },
+  { name: 'Maplibre GL JS', value: 'maplibre-gl' },
+];
+
+const typeToRenderers = {
+  google: [{ name: 'Google Maps', value: 'google' }],
+  leaflet: [{ name: 'Leaflet', value: 'leaflet' }],
+  tangram: [{ name: 'Tangram (Leaflet)', value: 'tangram' }],
+  'vector-gl': glVectorRenderers,
```

Comment:
Opening up to the possibility of having less specific `type` value in config files--rather than explicitly saying `mapbox-gl` or `maplibre-gl`, one should be able to say something more generic like `vector-gl`.